File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/playgames/GooglePlayGamesActivity.java
package com.jdroid.android.sample.ui.google.playgames;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class GooglePlayGamesActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return GooglePlayGamesFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/playgames/GooglePlayGamesActivity.kt
package com.jdroid.android.sample.ui.google.playgames

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class GooglePlayGamesActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return GooglePlayGamesFragment::class.java
    }
}
